sos/report/plugins/ceph.py

```diff
@@ -37,6 +38,12 @@ class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
         'ceph-osd@*'
     )
 
+    # This check will enable the plugin regardless of being
+    # containerized or not
+    files = (
+        '/etc/ceph/ceph.conf'
+    )
```

Comment:
Needs to be a tuple, at the moment it's a str.

Reply:
Fixed